db/CouriersDB.go

```diff
@@ -34,6 +34,22 @@ type SmallInfo struct {
 	Surname     string `json:"surname"`
 }
 
+func (r *CourierPostgres) SaveCourierInDB(courier *Courier) error {
+	db := ConnectDB()
+	defer db.Close()
+
+	insertValue := `INSERT INTO "couriers" ("name","ready to go","phone_number","email","rating","photo","surname","number of failures","deleted") VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9)`
```

Comment:
Shouldn't `number of failures be defaulted to 0 during insert? I don't think that new courier can have failures. Same for deleted. Both of them can be even defaulted on DB level, not in code.

Reply:
fixed